src/SDKs/RecoveryServices.SiteRecovery/RecoveryServices.SiteRecovery.Tests/RecoveryServices.SiteRecovery.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@shahabhijeet as discussed offline...we dont have dependency on Resource manager in our projects. Have removed all the stale references that we had.